client-app/core/composables/useWhiteLabeling.ts

```diff
@@ -0,0 +1,44 @@
+import { computed, shallowRef } from "vue";
+import { getGetWhiteLabelingSettings } from "@/core/api/graphql/whiteLabeling/queries";
+import { Logger, convertToExtendedMenuLink } from "@/core/utilities";
+import { useThemeContext } from "./useThemeContext";
+import type { WhiteLabelingSettingsType } from "@/core/api/graphql/types";
+
+const whiteLabelingSettings = shallowRef<WhiteLabelingSettingsType>();
+const { modulesSettings, themeContext } = useThemeContext();
+
+const MODULE_KEYS = {
+  ID: "VirtoCommerce.WhiteLabeling",
+  ENABLE_STATE: "WhiteLabeling.WhiteLabelingEnabled",
+};
+
+const moduleSettings = computed(() => modulesSettings.value?.find((module) => module.moduleId === MODULE_KEYS.ID));
+const moduleEnabled = computed(
+  () => moduleSettings.value?.settings?.find((item) => item.name === MODULE_KEYS.ENABLE_STATE)?.value,
+);
+
+export function useWhiteLabeling() {
```

Comment:
it's global and not reusable so better use createGlobalState like in useThemeContext and put all state inside the function

https://vueuse.org/shared/createGlobalState/